numpy/core/src/private/ufunc_override.c

```diff
@@ -46,12 +50,13 @@ normalize_reduce_args(PyUFuncObject *ufunc, PyObject *args,
     int i;
     PyObject *obj;
 
-    for (i = 0; i < nargs; i++) {
+    *normal_args = PyTuple_GetSlice(args, 0, 1);
+    for (i = 1; i < nargs; i++) {
         obj = PyTuple_GET_ITEM(args, i);
-        if (i == 0) {
-            *normal_args = PyTuple_GetSlice(args, 0, 1);
+        if (obj == Py_None) {
+            continue;
         }
-        else if (i == 1) {
+        if (i == 1) {
```

Comment:
Could even make a switch on nargs and complete by falling through in reverse order, thus avoiding the loop.